utility_network/configure-subnetwork-trace/src/main/resources/configure_subnetwork_trace/main.fxml

```diff
@@ -28,17 +28,20 @@
 <StackPane stylesheets="/configure_subnetwork_trace/style.css" xmlns="http://javafx.com/javafx/10.0.2-internal" xmlns:fx="http://javafx.com/fxml/1"
            fx:controller="com.esri.samples.configure_subnetwork_trace.ConfigureSubnetworkTraceController">
 
-    <VBox>
+    <VBox StackPane.alignment="TOP_LEFT">
         <HBox>
             <Label text="Trace options:"/>
             <CheckBox fx:id="includeBarriersCheckBox" text="Include Barriers"/>
             <CheckBox fx:id="includeContainersCheckBox" text="Include Containers"/>
         </HBox>
 
-        <HBox alignment="CENTER_LEFT">
-            <Label text="Define new conditions"/>
-            <ComboBox fx:id="comparisonSourcesComboBox" onAction="#onComparisonSourceChanged"/>
-            <ComboBox fx:id="comparisonOperatorsComboBox"/>
+        <HBox>
+            <Label text="Define new conditions:"/>
+        </HBox>
```

Comment:
What's the reason for separating this from the combo boxes?
Either way, since it's only one element, you can get rid of the outer HBox here and the behaviour would be the same:

```suggestion
        <Label text="Define new conditions:"/>
```